packages/vector_graphics_compiler/lib/src/paint.dart

```diff
@@ -567,7 +567,7 @@ class Stroke {
 class Fill {
   /// Creates a new immutable set of drawing attributes for a [Paint].
   const Fill({
-    this.color,
+    this.color = Color.opaqueBlack,
```

Comment:
My first reaction was that we can't do this because it'll break inheritence, but we no longer use fill for inheritence yayyyy